packages/react-button/src/components/Button/useButton.ts

```diff
@@ -1,4 +1,4 @@
-import { useImperativeHandle, useRef } from 'react';
```

Comment:
I suggest to avoid such imports as previously `React` was not imported to this scope however it was used in ` React.RefObject`